### PR TITLE
Remove enkf plot_config staff from ert

### PR DESCRIPTION
**Pull request by xjules**
**Task**
_Short description of the task_
PlotConfig has been remove from libres 
here: https://github.com/equinor/libres/pull/560
We need to update PlotSettings and fix tests to work without enkf plot_config

**Pre un-WIP checklist**
- [ ] Equinor tests pass locally
- [ ] Have completed graphical integration test steps

**Depends on**
* Equinor/libres#
* Equinor/libecl#

### CODE REVIEW

**Comment by lars-petter-hauge** [comment]
Ref failing tests, rebase master and you're good to go, fixed in https://github.com/equinor/ert/pull/383/commits/729a2b0bae822382e6c016a71e42006c2d2dc69e

**Comment by xjules** [comment]
> Ref failing tests, rebase master and you're good to go, fixed in [729a2b0](https://github.com/equinor/ert/commit/729a2b0bae822382e6c016a71e42006c2d2dc69e)

Ok, rebased and the tests have passed.

File: ert_gui/plottery/plot_config.py [hunk 1]
```diff
@@ -7,11 +7,12 @@ class PlotConfig(object):
     # The plot_settings input argument is an internalisation of the (quite few) plot
     # policy settings which can be set in the configuration file.
     
-    def __init__(self, plot_settings, title="Unnamed", x_label=None, y_label=None):
+    def __init__(self, plot_settings=None, title="Unnamed", x_label=None, y_label=None):
```
Comment (ManInFez) [inline ert_gui/plottery/plot_config.py]: If plot_settings are not used anymore, could it not be removed from the parameter-list?

Comment (xjules) [reply ert_gui/plottery/plot_config.py]: I left it there in order it can be set from code if required.

**Comment by markusdregi** [comment]
@xjules Could you bump the _libres_ version also to verify that it now builds against _libres_ after the plot config was removed?

**Comment by lars-petter-hauge** [comment]
I suppose we need to actually run ERT as well, I don’t think there are tests which would cover this.

**Comment by xjules** [comment]
> @xjules Could you bump the _libres_ version also to verify that it now builds against _libres_ after the plot config was removed?

Sure

**Pull request merged**